src/labels/CustomLabels.labels

```diff
@@ -1617,6 +1617,30 @@
         <shortDescription>Error when the functionality is not supported for Elevate Recurring Donation.</shortDescription>
         <value>This functionality is currently not supported for Recurring Donations generated from Elevate Payment Services.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevateDisableButtonLabel</fullName>
```

Comment:
@sam-knox , can you please verify three new labels applicable to RD2 entry form explicitly?